bot.js

```diff
@@ -285,7 +285,7 @@ function rankOrEmpty(name, obj) {
     if (!obj || !obj.tier) {
         return '';
     }
-    return name + emojiForTier(obj.tier) + ' , div ' + (obj.division + 1) + '\n';
+    return name + emojiForTier(obj.tier) + ' , div ' + (obj.division + 1) + ": " + obj.rankPoints + '\n';
```

Comment:
Nice addition! 
my only suggestion would be formatting as "(ddd)" or maybe "(ddd mmr)" or "(ddd rating)"
such a shame JS doesn't have sprintf too...